### PR TITLE
core.thread.osthread: Remove version ExternStackShell

### PR DESCRIPTION
**Pull request by ibuclaw**
This implementation for PPC64 does not work

### CODE REVIEW

**Comment by ibuclaw** [comment]
Ping @kinke

**Comment by ibuclaw** [comment]
It looks like LDC already ignored this. :-(

https://github.com/ldc-developers/druntime/blob/f208b24a2553978cbf3b7333537f75f0ee6bca99/src/core/thread/osthread.d#L2548-L2549

**Comment by Geod24** [comment]
Well it was introduced by @redstar in 2013 : https://github.com/dlang/druntime/commit/a8ea1a5dd0c6d58fe5ac9798fc04f3bce51696aa

**Comment by ibuclaw** [comment]
> Well it was introduced by @redstar in 2013 : [a8ea1a5](https://github.com/dlang/druntime/commit/a8ea1a5dd0c6d58fe5ac9798fc04f3bce51696aa)

And then removed from LDC in 2014.

File: src/core/threadasm.S [hunk 1]
```diff
@@ -27,153 +27,7 @@
...
-    blr
-    .long 0
-    .quad 0
-.Lend:
-    .size _D4core6thread18callWithStackShellFNbMDFNbPvZvZv, .Lend-.L._D4core6thread18callWithStackShellFNbMDFNbPvZvZv
-    .cfi_endproc
-
-#elif defined( __ppc__ ) || defined( __PPC__ ) || defined( __powerpc__ )
-
+#if (defined( __PPC__ ) || defined( __powerpc__ )) && !defined( __PPC64__ )
```
Comment (kinke) [inline src/core/threadasm.S:30]: Previously, `defined( __ppc__ )` was checked too.

Comment (ibuclaw) [reply src/core/threadasm.S:30]: Who defines it?  It's non-standard, hence I threw it out.

Comment (kinke) [reply src/core/threadasm.S:30]: I don't know, it just stood out as a potentially unwanted change.

**Comment by ibuclaw** [comment]
@kinke - I've left the #ifdefs alone, and just replaced the contents of `__PPC64__` with a TODO comment.

**Comment by ibuclaw** [comment]
There are two approvals + tests pass -> merge.

**Comment by denizzzka** [comment]
Some days ago I used it to implement my own external `callWithStackShell`. Huh

**Pull request merged**